src/OrleansBenchmarks/OrleansBenchmarks.csproj

```diff
@@ -0,0 +1,111 @@
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Serialization\PocoState.cs" />
+    <Compile Include="Serialization\SerializationBenchmarkConfig.cs" />
+    <Compile Include="MapReduce\MapReduceBenchmarkConfig.cs" />
+    <Compile Include="MapReduce\MapReduceBenchmark.cs" />
+    <Compile Include="Program.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Serialization\SerializationBenchmarks.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="App.config" />
+    <None Include="project.json" />
+  </ItemGroup>
+  <ItemGroup>
+    <Analyzer Include="..\packages\Microsoft.CodeAnalysis.Analyzers.1.1.0\analyzers\dotnet\cs\Microsoft.CodeAnalysis.Analyzers.dll" />
+    <Analyzer Include="..\packages\Microsoft.CodeAnalysis.Analyzers.1.1.0\analyzers\dotnet\cs\Microsoft.CodeAnalysis.CSharp.Analyzers.dll" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\test\Tester\Tester.csproj">
+      <Project>{a9c8ffec-4947-4f04-ba73-1f17b329a55d}</Project>
+      <Name>Tester</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\BenchmarkGrainInterfaces\OrleansBenchmarkGrainInterfaces.csproj">
+      <Project>{936397df-8d11-4244-a493-767d31f9d727}</Project>
+      <Name>OrleansBenchmarkGrainInterfaces</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\BenchmarkGrains\OrleansBenchmarkGrains.csproj">
+      <Project>{071cb148-eaa3-4f85-a1f7-52d1b7fdaf82}</Project>
+      <Name>OrleansBenchmarkGrains</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\ClientGenerator\ClientGenerator.csproj">
+      <Project>{e782dd19-51f7-4f66-8217-bacac33767e4}</Project>
+      <Name>ClientGenerator</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\OrleansHost\OrleansHost.csproj">
+      <Project>{5c177f58-a40c-449f-8c2f-a2657f963edc}</Project>
+      <Name>OrleansHost</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\OrleansRuntime\OrleansRuntime.csproj">
+      <Project>{6ff2004c-cdf8-479c-bf27-c6bfe8ef93e0}</Project>
+      <Name>OrleansRuntime</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\Orleans\Orleans.csproj">
+      <Project>{bc1bd60c-e7d8-4452-a21c-290aec8e2e74}</Project>
+      <Name>Orleans</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="BenchClientConfiguration.xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+    <Content Include="BenchServerConfiguration.xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="..\packages\Microsoft.Diagnostics.Tracing.TraceEvent.1.0.41\build\Microsoft.Diagnostics.Tracing.TraceEvent.targets" Condition="Exists('..\packages\Microsoft.Diagnostics.Tracing.TraceEvent.1.0.41\build\Microsoft.Diagnostics.Tracing.TraceEvent.targets')" />
```

Comment:
these are probably a remnant of before migrating to project.json, and I believe it's the reason why it's failing the build. Search for `..\packages\` and you'll see several instances